releasenotes/notes/add-classical-predecessors-9ecef0561822e934.yaml

```diff
@@ -0,0 +1,23 @@
+---
+features:
+  - |
+    Added two methods, classical_predecessors and classical_successors an alternative to select the classical wires without having to go to the 
+    inner graph object directly of a node in the DAG.
+
+    The following example illustrates the new functionality::
+
+        from qiskit import QuantumCircuit
+        from qiskit import QuantumCircuit, QuantumRegister, ClassicalRegister
+        from qiskit.converters import circuit_to_dag
+
+        q = QuantumRegister(3, 'q')
+        c = ClassicalRegister(3, 'c')
+        circ = QuantumCircuit(q, c)
+        circ.h(q[0])
+        circ.cx(q[0], q[1])
+        circ.measure(q[0], c[0])
+        circ.rz(0.5, q[1]).c_if(c, 2)
+        dag = circuit_to_dag(circ)
+        for i in range(len(dag.op_nodes())):
+            dag.classical_predecessors(dag.node(i))
+            dag.classical_successors(dag.node(i))
```

Comment:
I would add some prints and output, to help illustrating the result.

Reply:
it-s done the changes @1ucian0  I did the changes